lib/jobs/ssh-job.js

```diff
@@ -37,6 +37,7 @@ function sshJobFactory(
         this.nodeId = this.context.target;
         this.commandUtil = new CommandUtil(this.nodeId);
         this.commands = this.commandUtil.buildCommands(options.commands);
+        this.sshExecOptions = options.sshExecOptions;
```

Comment:
@brianparry 's team has added a "ibms" (InBand Management, just like obms) document, the ssh settings will be stored as a kind of ibm. So the ssh setting should be fetched from node's ibms settings.

Add @brianparry @BillyAbildgaard @dalebremner for comment.